index.js

```diff
@@ -1,13 +1,21 @@
 /** @type {import('eslint').Linter.Config} */
 module.exports = {
   extends: ['seek'],
-  ignorePatterns: ['**/.eslintrc.js'],
+  ignorePatterns: [
+    '**/.eslintrc.js',
+
+    // Gantry resource files support non-standard template syntax
```

Comment:
What's _standard_ yaml template syntax?

Reply:
Sorry for the confusion 🙂

```suggestion
    // Gantry resource files support non-standard syntax (Go templating)
```